test/jdk/java/awt/TrayIcon/ActionEventTest/ActionEventTest.java

```diff
@@ -26,16 +26,12 @@
  * @bug 6191390 8154328
  * @key headful
  * @summary Verify that ActionEvent is received with correct modifiers set.
- * @modules java.desktop/java.awt:open
- * @modules java.desktop/java.awt.peer
- * @library /lib/client ../
- * @library /java/awt/patchlib
- * @build java.desktop/java.awt.Helper
- * @build ExtendedRobot SystemTrayIconHelper
```

Comment:
SystemTrayIconHelper uses ExtendedRobot (robot.glide(), click()) so I think you might still need it for this test if this PR is being integrated before  https://github.com/openjdk/jdk/pull/22044

Reply:
If the SystemTrayIconHelper itself is building ExtendedRobot do I still need to build ER on ActionEventTest?

Reply:
reverting this test